packages/cashier/src/components/cashier-locked/cashier-locked-provider.tsx

```diff
@@ -0,0 +1,368 @@
+                        components={[
+                            <a
+                                key={0}
+                                className='link'
+                                rel='noopener noreferrer'
+                                href={'/account/proof-of-identity'}
+                            />,
+                            <a key={1} className='link' rel='noopener noreferrer' href={'/account/proof-of-address'} />,
+                        ]}
+                    />
+                ),
+            };
+        if (ask_fix_details)
+            return {
+                icon: 'IcCashierWithdrawalLock',
+                title: localize('Withdrawals are locked'),
+                description: (
+                    <Localize
+                        i18n_default_text='Your <0>personal details</0> are incomplete. Please go to your account settings and complete your personal details to enable withdrawals.'
+                        components={[
+                            <a key={0} className='link' rel='noopener noreferrer' href={'/account/personal-details'} />,
+                        ]}
+                    />
+                ),
+            };
+        if (withdraw_service_unavailable_for_pa)
+            return {
+                icon: 'IcCashierWithdrawalLock',
+                title: localize('Withdrawals are locked'),
+                description: localize('This feature is not available for payment agents.'),
+            };
+        if (no_withdrawal_or_trading_status)
+            return {
+                icon: 'IcCashierWithdrawalLock',
+                title: localize('Withdrawals are locked'),
+                description: localize(
+                    'Unfortunately, you can only make deposits. Please contact us via live chat to enable withdrawals.'
+                ),
+            };
+        if (withdrawal_locked_status)
+            return {
+                icon: 'IcCashierWithdrawalLock',
+                title: localize('Withdrawals are locked'),
+                description: localize(
+                    'Unfortunately, you can only make deposits. Please contact us via live chat to enable withdrawals.'
+                ),
+            };
+        if (only_pa_withdrawals_allowed_status)
+            return {
+                icon: 'IcCashierWithdrawalLock',
+                title: localize('Withdrawals are locked'),
+                description: localize(
+                    'You can only make deposits. Please contact us via live chat for more information.'
+                ),
+            };
+    }
+
+    return {
+        icon: 'IcCashierLocked',
+        title: localize('Cashier is locked'),
+        description: localize(
+            'Your cashier is currently locked. Please contact us via live chat to find out how to unlock it.'
+        ),
+    };
+};
+
+export default {
+    getMessage,
+};
```

Comment:
```suggestion
export default { getMessage };
```

Reply:
Can't export like this? As there is nothing else to be exported from this file.

```suggestion
export default getMessage;
```

Reply:
@hamid-deriv It was just for better readability, So we could have something like `provider.getMessage` instead of `getMessage` which was more clear what was happening. I will update the code 🙇🏻